packages/starlight/user-components/LinkCard.astro

```diff
@@ -1,20 +1,36 @@
 ---
 import Icon from './Icon.astro';
+import Badge from '../components/Badge.astro';
+import type { Badge as BadgeType } from '../schemas/badge';
+import { BadgeConfigSchema } from '../schemas/badge';
```

Comment:
```suggestion
import { BadgeConfigSchema, type BadgeUserConfig } from '../schemas/badge';
import { errorMap } from '../utils/error-map';
```

We can consolidate both `import` here and I explain `BadgeUserConfig` and the `errorMap` a little bit below.